### PR TITLE
 🚧 A thin plate of glass

### PR DESCRIPTION
**Pull request by m-kuhn**
In a frame is called a window. Plural: windows.
You can look through it and see that there's more on the other side.

### CODE REVIEW

File: src/qml/qgismobileapp.qml [hunk 1]
```diff
@@ -1383,6 +1383,7 @@ ApplicationWindow {
       }
     }
     onDropped: {
+      console.warn('Drop URL: ' + drop.urls[0])
```
Comment (m-kuhn) [inline src/qml/qgismobileapp.qml]: Remove

File: src/qml/imports/Theme/Theme.qml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 pragma Singleton
 
-import QtQuick 2.12
+import QtQuick 2.11
```
Comment (m-kuhn) [inline src/qml/imports/Theme/Theme.qml]: @3nids @nirvn do you see any problem with downgrading these imports to 2.11?

File: src/qml/WelcomeScreen.qml [hunk 3]
```diff
@@ -86,8 +86,10 @@ Page {
         Button {
           id: localProjectButton
           padding: 8 * dp
+/* Not available on Qt 5.11 (OSGeo4W)
```
Comment (m-kuhn) [inline src/qml/WelcomeScreen.qml]: @nirvn do you see a problem here?

Comment (nirvn) [reply src/qml/WelcomeScreen.qml]: Could you try to adjust the padding or margin values to achieve same size?

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e11d29e28a7442628398db4c43f595a1b25a7190-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e11d29e28a7442628398db4c43f595a1b25a7190-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e11d29e28a7442628398db4c43f595a1b25a7190-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e11d29e28a7442628398db4c43f595a1b25a7190-x86_64.apk)

File: src/qml/WelcomeScreen.qml (hunk 3 above)
Comment (m-kuhn) [reply src/qml/WelcomeScreen.qml]: I suspect an update soon tbh https://lists.osgeo.org/pipermail/qgis-developer/2020-January/060067.html

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-a1b3f7b1f9b40bd47dad6407e02f035c303072df-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-a1b3f7b1f9b40bd47dad6407e02f035c303072df-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-a1b3f7b1f9b40bd47dad6407e02f035c303072df-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-a1b3f7b1f9b40bd47dad6407e02f035c303072df-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dbbb7ad70ab6a50b2559e9b97e7348e046a61e55-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dbbb7ad70ab6a50b2559e9b97e7348e046a61e55-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dbbb7ad70ab6a50b2559e9b97e7348e046a61e55-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dbbb7ad70ab6a50b2559e9b97e7348e046a61e55-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-98fc277bea1ff16174d916463cff53bf41daaff5-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-98fc277bea1ff16174d916463cff53bf41daaff5-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-98fc277bea1ff16174d916463cff53bf41daaff5-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-98fc277bea1ff16174d916463cff53bf41daaff5-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-ea347a9d455c845c9cdfff91d01675c6e2f31e05-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-ea347a9d455c845c9cdfff91d01675c6e2f31e05-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-ea347a9d455c845c9cdfff91d01675c6e2f31e05-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-ea347a9d455c845c9cdfff91d01675c6e2f31e05-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b2103a2aac6beacedea6097cc9ea8776c3514b30-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b2103a2aac6beacedea6097cc9ea8776c3514b30-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b2103a2aac6beacedea6097cc9ea8776c3514b30-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b2103a2aac6beacedea6097cc9ea8776c3514b30-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-d6c0cd21104c3f366d0436f96ac5326f16038a54-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-d6c0cd21104c3f366d0436f96ac5326f16038a54-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-d6c0cd21104c3f366d0436f96ac5326f16038a54-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-d6c0cd21104c3f366d0436f96ac5326f16038a54-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-1ee91838bf8fb24452483681ca1b2b9b2aa7514c-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-1ee91838bf8fb24452483681ca1b2b9b2aa7514c-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-1ee91838bf8fb24452483681ca1b2b9b2aa7514c-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-1ee91838bf8fb24452483681ca1b2b9b2aa7514c-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5849c74a58adc620c3ba8bbfed2f37452d61bdaa-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5849c74a58adc620c3ba8bbfed2f37452d61bdaa-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5849c74a58adc620c3ba8bbfed2f37452d61bdaa-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5849c74a58adc620c3ba8bbfed2f37452d61bdaa-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5e37d5f94033011ff5dd10a88bcba88a823bd2dd-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5e37d5f94033011ff5dd10a88bcba88a823bd2dd-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5e37d5f94033011ff5dd10a88bcba88a823bd2dd-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-5e37d5f94033011ff5dd10a88bcba88a823bd2dd-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-f7f443fa12004962c0f1275f9bbcc7567caa50e9-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-f7f443fa12004962c0f1275f9bbcc7567caa50e9-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-f7f443fa12004962c0f1275f9bbcc7567caa50e9-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-f7f443fa12004962c0f1275f9bbcc7567caa50e9-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e88f471b023a42bfb79e3b55616d5262a7359925-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e88f471b023a42bfb79e3b55616d5262a7359925-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e88f471b023a42bfb79e3b55616d5262a7359925-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-e88f471b023a42bfb79e3b55616d5262a7359925-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-28d5e38279900f219763be43b91495dc641ebdbf-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-28d5e38279900f219763be43b91495dc641ebdbf-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-28d5e38279900f219763be43b91495dc641ebdbf-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-28d5e38279900f219763be43b91495dc641ebdbf-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b34223629a4be3b7448c3e5af9a8c8e288e05134-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b34223629a4be3b7448c3e5af9a8c8e288e05134-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b34223629a4be3b7448c3e5af9a8c8e288e05134-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-b34223629a4be3b7448c3e5af9a8c8e288e05134-x86_64.apk)

File: .azure-pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    include:
+    - master
+
+pr:
+- master
+
+jobs:
+- job: build
+  pool:
+    vmImage: vs2015-win2012r2
```
Comment (m-kuhn) [inline .azure-pipeline.yml]:
```suggestion
    vmImage: vs2017-win2016
```

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-42086c01497d82e1e743955fe6b1533bd075a76b-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-42086c01497d82e1e743955fe6b1533bd075a76b-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-42086c01497d82e1e743955fe6b1533bd075a76b-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-42086c01497d82e1e743955fe6b1533bd075a76b-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-9f9c860a1add34b659f7468e283bdd2351159f9b-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-9f9c860a1add34b659f7468e283bdd2351159f9b-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-9f9c860a1add34b659f7468e283bdd2351159f9b-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-9f9c860a1add34b659f7468e283bdd2351159f9b-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dec72db6785dbec955cdc7e98d11296d8989365f-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dec72db6785dbec955cdc7e98d11296d8989365f-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dec72db6785dbec955cdc7e98d11296d8989365f-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-793-dec72db6785dbec955cdc7e98d11296d8989365f-x86_64.apk)

**Pull request closed without merging**